Docs/reference/data/mongodb.toml

```diff
@@ -1,5 +1,5 @@
 githubRepo = "mongo-csharp-driver"
 githubBranch = "master"
-currentVersion = "2.9"
+currentVersion = "2.10"
```

Comment:
I'm honestly unsure if this should be 2.9, since our "currentVersion" is 2.9.2. Previewing both the landing page and the api docs page in hugo, things look okay, but I don't know of a way to preview both using hugo?

Reply:
Java driver also uses the "short release name" here, so I think "2.9" was correct. 

What is the affect of changing this to 2.10?

Reply:
![image](https://user-images.githubusercontent.com/33040432/66154162-e181d100-e5ea-11e9-83d7-8262cc96e28a.png)

Reply:
This changes the version in the sidebar to 2.10

Reply:
Which sidebar exactly?

Reply:
Never mind, I see it.